docs/user-guide/networkpolicies.md

```diff
@@ -0,0 +1,82 @@
+---
+---
+
+* TOC
+{:toc}
+
+## What is a _Network Policy_?
+
+A Network Policy is a specification of how groupings of pods are allowed to communicate with each other and other network endpoints.
+
+NetworkPolicy resources use labels to select pods and define whitelist rules which allow traffic to the selected pods in addition to what is allowed by the ingress isolation policy for a given namespace.
```

Comment:
@pwittrock - There is no relationship between ingress as used in a `NetworkPolicy` and `Ingress` the API object.  I'll try to make that more clear.